pkg/agents/team1/agent1/agent.go

```diff
@@ -0,0 +1,48 @@
+package agent1_1
+
+import (
+	"log"
+
+	"github.com/SOMAS2021/SOMAS2021/pkg/agents"
+	"github.com/SOMAS2021/SOMAS2021/pkg/infra/messages"
+	"github.com/SOMAS2021/SOMAS2021/pkg/utils/abm"
+)
+
+type CustomAgent1 struct {
+	*agents.Base
+	myNumber int
```

Comment:
What is the point of myNumber? Better naming would clarify. Seems like an ID but not obvious.

Reply:
Example of a custom parameter you can add in